janusgraph-core/src/test/java/org/janusgraph/graphdb/tinkerpop/optimize/step/fetcher/MultiQueriableStepBatchFetcherTest.java

```diff
@@ -46,7 +48,7 @@ public void setupMocks(){
         Mockito.doReturn(traversal).when(traversal).asAdmin();
         Mockito.doReturn(Optional.of(tx)).when(traversal).getGraph();
         Mockito.doReturn(true).when(tx).isOpen();
-        Mockito.doReturn(multiQuery).when(tx).multiQuery();
+        Mockito.doReturn(multiQuery).when(tx).multiQuery(anyCollection());
```

Comment:
**💭 thought:** When not specifying the `multiQuery()` argument type, the mocked method return‏s null (in my tests). Could it be because that method is overloaded and accepts both a Collection and varargs?